lib/proxy/util.rb

```diff
@@ -91,6 +91,29 @@ def escape_for_shell(command)
     end
   end
 
+  def shell_command(cmd, wait = true)
```

Comment:
Could you open another PR to update puppetrun providers to use this method instead the one defined in modules/puppet_proxy_common/runner.rb?